examples/Hermitian_Phasing.py

```diff
@@ -81,7 +81,7 @@
         data.setaxis("t2", lambda x: x - rough_center).register_axis({"t2": 0})
         data.ft("t2")
         data.ift("t2")
-        data /= zeroth_order_ph(select_pathway(data,signal_pathway))
+        data /= zeroth_order_ph(select_pathway(data,signal_pathway), fl=fl)
```

Comment:
here, I just added the diagnostic plot, actually, so you just need to make sure the plot number is set correctly for sphinx.